src/_includes/layouts/page.njk

```diff
@@ -17,17 +17,22 @@
     </div>
   </article>
   <div class="homepage-cards">
-    <a href="/initiatives/" class="blue card">
-      <h2>Read current views in inclusive data science</h2>
+    <a href="/resources/" class="blue card">
+      <h2>Resource Library</h2>
+      <p>Search IDRC's library of AI and data analytics resources</p>
     </a>
-    <a href="/events/" class="yellow card">
-      <h2>Participate in our inclusion challenge initiatives</h2>
+    <a href="/events/" class="green card">
+      <h2>Events</h2>
+      <p>Find out about IDRC events and how you can participate</p>
     </a>
-    <a href="/badges/" class="azure card">
-      <h2>Earn We Count Badges</h2>
+    <a href="/initiatives/" class="azure card">
+      <h2>Inclusive AI Initiatives</h2>
+      <p>Explore IDRC project activities</p>
     </a>
-    <a href="/resources/" class="green card">
-      <h2>Search the We Count Library</h2>
+    <a href="/rtdd/" class="yellow card">
+      <h2>Regulating the Digital</h2>
+      <h2 class="card-title__french">Réguler le numérique</h2>
+      <p>Addressing digital exclusion / Lutter contre l’exclusion numérique</p>
```

Comment:
Nesting both the card title and description text in the link leads to excessive and hard to understand announcements for screen reader users. It's also not good to have two H2s one after another in the RTDD card, because the semantics of the element mean that it should be the title to a section of content, and there's no content between them. It's also important to properly tag content in a different language than the rest of the site, so the French text should have a `lang="fr"` tag. Let's discuss tomorrow.

Reply:
```html
<div class="card">
<h2>Regulating the Digital<br />
<span lang="fr">Réguler le numérique</span></h2>
</div>
```

```css
.card h2 [lang="fr"] {

}
```

Reply:
```html
<div class="card">
<h2>Regulating the Digital<br />
<span lang="fr">Réguler le numérique</span></h2>
</div>
```

```css
.card h2 [lang="fr"] {

}
```